### ISSUE TITLE
Tmux passthrough error when opts enabled in Lazyvim 

### ISSUE THREAD

**Issue by pchest** [3rd/image.nvim#254]
## Issue

When I have "opts = {}" enabled in Lazyvim, even with no specific options present, I encounter an error in tmux whenever I open a file. This is true even if I have `set -g allow-passthrough on` in my .tmux.conf file. 

## Replication

_All options disabled_

![image.nvim configured with empty options](https://github.com/user-attachments/assets/8ea2bf96-1189-43f8-8fbc-1dcfa041155b)

_Error message_

![Error message when opening any file in tmux](https://github.com/user-attachments/assets/f191de5e-3b73-4687-87a1-aba03dfd9ee8)

## System Info
OS: pop os 22.04
tmux version: 3.5
nvim version: 0.10.2

**Comment by b0o** [comment]
Did you reload your tmux config after setting the option?

**Comment by pchest** [comment]
@b0o Yes, I did. This error persists after refreshing the terminal or even a full system reboot. I also encountered it on another device with pop os 22.04 installed.

**Comment by skogler** [comment]
@pchest  I think this is related to tmux >= 3.3, I experimented a bit and changed the tmux option to:
```
set -pg allow-passthrough on
```

**Comment by pchest** [comment]
@skogler Strangely, that doesn't seem to resolve the issue. I changed my tmux.conf, but still got a similar error when image.nvim options are enabled.

## .tmux.conf
![image](https://github.com/user-attachments/assets/d0c6484a-a8b3-49b6-8feb-b63e566f7fb9)

## Error message

![image](https://github.com/user-attachments/assets/2de8ce46-436a-4e95-8d6d-296d16f85327)

**Comment by skogler** [comment]
@pchest Ah, sorry, I think my tmux was not reloaded properly. I also got it again now on this boot.

Changing it to:

```
setw -g allow-passthrough on
set -g visual-activity off
```

Seems to have resolved it though, but not 100% sure if thats why I don't get the warning anymore now

**Comment by pchest** [comment]
Thanks for the reply! @skogler. Unfortunately, this change doesn't seem to do the trick either. 

![image](https://github.com/user-attachments/assets/2b467336-19b1-4ff6-8d9e-a6d748e6812e)

**Comment by willgraham345** [comment]
For those that saw this, it worked for me to have tmux 3.5a installed. That fixed my problem here.

**Comment by pchest** [comment]
@3rd I haven't seen the error since your reply, so that fix did the trick. Thanks!
